<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP_Socket_Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="title">
     <string>服务器配置</string>
    </property>
    <widget class="QWidget" name="layoutWidgetServer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutServer">
      <item row="0" column="0">
       <widget class="QLabel" name="labelIPAddress">
        <property name="text">
         <string>IP地址：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxIPAddress"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPort">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>21</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelUser">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditUser">
        <property name="text">
         <string>admin</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="text">
         <string>qt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFile">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>250</width>
      <height>180</height>
     </rect>
    </property>
    <property name="title">
     <string>文件配置</string>
    </property>
    <widget class="QWidget" name="layoutWidgetFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutFile">
      <item row="0" column="0">
       <widget class="QLabel" name="labelRootPath">
        <property name="text">
         <string>文件地址：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditRootPath"/>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="toolButtonRootPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="checkBoxAllowOneIP">
        <property name="text">
         <string>只允许一个IP登录</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxOnlyRead">
        <property name="text">
         <string>只读</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QCheckBox" name="checkBoxAllowAnyOneLogin">
        <property name="text">
         <string>允许匿名用户</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxOnlyWrite">
        <property name="text">
         <string>只写</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonStartServer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>15</y>
      <width>100</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>开启服务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopServer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>100</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>停止服务</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetIPAddress">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>100</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteIP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBoxIPAddress</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
